packages/runner/lib/sdk/sdk.ts

```diff
@@ -116,11 +116,14 @@ export class NangoActionRunner extends NangoActionBase<never, Record<string, str
                 return connection;
             },
             getIntegrationConfig: async () => {
-                const integration = await configService.getProviderConfig(this.providerConfigKey, this.environmentId);
-                if (integration) {
+                const integration = await this.getIntegration({ include: ['credentials'] });
```

Comment:
I wouldn't be against revisiting this design decision to grab the integration config when executing a proxy request from the runner. It adds a extra request to Nango API and put the server as a dependency for oauth1 proxy request from the Runner, which we always tried to avoid. if integration credentials must be available in the runner they should be passed as part of the Nango props. What do you think @hassan254-prog ? I should have commented about it in your previous PR.